### DISCUSSION TITLE
Mikrotik RouterOS CHR VM

### DISCUSSION THREAD

**Discussion by mdarko** [tteck/Proxmox#3347] [General]
Greeting,

Maybe instead URL=https://download.mikrotik.com/routeros/7.12.1/chr-7.12.1.img.zip

add the latest version

Stable:
https://upgrade.mikrotik.com/routeros/NEWESTa7.stable
(at time I write: "7.15.2 1719402157\n")

Testing:
https://upgrade.mikrotik.com/routeros/NEWESTa7.testing
(at time I write: "7.16beta4 1719924461\n")

Development:
https://upgrade.mikrotik.com/routeros/NEWESTa7.development
(at time I write: "7.16beta4 1719924461\n")

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/commit/22c7801217482eb9f0ecfc37f2c0ebe6cdf14351

**Reply by mdarko** [reply]
STABLE=$(curl -s https://upgrade.mikrotik.com/routeros/NEWESTa7.stable | awk '{print $1 }')
URL=[https://download.mikrotik.com/routeros/${STABLE}/chr-${STABLE}.img.zip](https://download.mikrotik.com/routeros/$%7BSTABLE%7D/chr-$%7BSTABLE%7D.img.zip)
